src/opensles/AudioStreamOpenSLES.cpp

```diff
@@ -76,7 +76,7 @@ SLuint32 AudioStreamOpenSLES::channelCountToChannelMaskDefault(int channelCount)
 
 static bool s_isLittleEndian() {
     static uint32_t value = 1;
-    return *((uint8_t *) &value) == 1; // Does address point to LSB?
+    return (*reinterpret_cast<uint8_t *>(&value) == 1);
```

Comment:
Please restore the comment.

Reply:
done, thanks! ( In old days, comment on the "return..." statement caused compile errors, I was under that impression.)  pointer is a special kind of value, once value is LE, pointer should be LE already, correct?

Reply:
The pointer is not little endian. It points to a value that is either big or little endian. This is a trick for querying endianness.